decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -628,6 +628,39 @@ def generate_localized_title(field = nil, skip_injection: false)
     organization
   end
 
+  factory :taxonomy, class: "Decidim::Taxonomy" do
+    name { Decidim::Faker::Localized.literal(generate(:name)) }
+    organization
+    parent { nil }
+    weight { nil }
+
+    trait :with_parent do
+      association :parent, factory: :taxonomy
+    end
+
+    trait :with_children do
+      transient do
+        children_count { 3 }
+      end
+
+      after(:create) do |taxonomy, evaluator|
+        create_list(:taxonomy, evaluator.children_count, parent: taxonomy, organization: taxonomy.organization)
+        taxonomy.reload
+        taxonomy.update(weight: taxonomy.children.count)
+      end
+    end
+  end
+
+  factory :taxonomy_filter, class: "Decidim::TaxonomyFilter" do
+    taxonomy
+    filterable { association(:proposal_component) }
```

Comment:
I would not create more bindings to `proposals` engine. 
```suggestion
    filterable { association(:dummy_component) }
```